source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -94,50 +124,46 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
         contentView.textField.backgroundColor = ColorUtils.hoverColorOnMouseExit()
     }
     
-    var value: String {
-        guard let selectedDate = selectedDate else {
-            return ""
-        }
-        return dateFormatter.string(from: selectedDate)
-    }
-    
-    var key: String {
-        guard let id = idString else {
-            logError("ID must be set on creation")
-            return ""
-        }
-        return id
-    }
-    
-    var isValid: Bool {
-        return !isHidden && !(superview?.isHidden ?? false)
-    }
-    
     // MARK: Actions
     @objc private func handleButtonAction() {
         let frame = isTimeMode ? NSRect(x: 0, y: 0, width: 122, height: 122) : NSRect(x: 0, y: 0, width: 138, height: 147)
         if let dateValue = selectedDate {
-            datepicker.dateValue = dateValue
+            datepickercalendar.dateValue = dateValue
+            datepickertextfield.dateValue = dateValue
+        } else {
+            datepickercalendar.dateValue = Date()
+            datepickertextfield.dateValue = Date()
```

Comment:
I see you're setting it here. 
Hmm, So this means that if `selectedDate` is wantedly set to nil, this will assume current date as the value.
I think this assumption can be ignored here.